### ISSUE TITLE
How would one adapt this to perform interior instead of exterior expansions?

### ISSUE THREAD

**Issue by jdcolburn** [maroba/multipoles#2]
As stated in the title. I'm interested in expanding distributions from a central point r onto a spherical interior. Any help would be appreciated.

**Comment by dmadisetti** [comment]
Yes, the $\frac{1}{r^(l+1)}$ term becomes $r^l$ and the $r'^l$ term becomes $\frac{1}{r'^(l+1)}$

**Comment by dmadisetti** [comment]
See https://github.com/cemel-jhu/multipoles/pull/1

**Comment by dmadisetti** [comment]
Closed by #6
